README.md

```diff
@@ -4,3 +4,5 @@
 
 The provided sophia examples or library functions are only tested to the extend represented. None of the code was security audited by aeternity or is provided to be used in production, without thorough checks.
 
+## Libraries
```

Comment:
this is an example, right?

Reply:
Shall I make new directory `examples` and move the project there?

Reply:
yes please :)